proto/ibc/core/channel/v1/tx.proto

```diff
@@ -276,23 +276,3 @@ message MsgAcknowledgementResponse {
 
   ResponseResultType result = 1;
 }
-
```

Comment:
Moved multihop message types into commitment.proto. Is this a good place for it?